src/Uno.UI/UI/Xaml/Window.Skia.cs

```diff
@@ -45,7 +45,7 @@ public void Init()
 			CoreWindow.SetInvalidateRender(QueueInvalidateRender);
 		}
 
-		internal Action InvalidateRender;
+		internal static Action InvalidateRender;
```

Comment:
Codacy found an issue: [Change the visibility of 'InvalidateRender' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932478)